test/ref/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+#####################################################################################
+# The MIT License (MIT)
+#
+# Copyright (c) 2015-2022 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#####################################################################################
+
+file(GLOB VERIFY_REF CONFIGURE_DEPENDS *.cpp)
+
+add_executable(test_ref ${VERIFY_REF})
+add_dependencies(tests test_ref)
+add_dependencies(check test_ref)
+target_link_libraries(test_ref migraphx migraphx_onnx migraphx_ref)
+target_include_directories(test_ref PUBLIC ../include)
```

Comment:
Rest of these lines should already be handles by `add_test_executable`.

Reply:
I switched to use `add_test_executable` but without `target_include_directories(test_ref PUBLIC ../include)`, it complains that it could not find `test.hpp`. So I have kept `target_include_directories(test_ref PUBLIC ../include)`.